cmd/skopeo/inspect.go

```diff
@@ -97,7 +99,13 @@ var inspectCmd = cli.Command{
 			outputData.Name = dockerImg.SourceRefFullName()
 			outputData.RepoTags, err = dockerImg.GetRepositoryTags()
 			if err != nil {
-				return fmt.Errorf("Error determining repository tags: %v", err)
+				// some registries may decide to block the "list all tags" endpoint
+				// gracefully allow the inspect to continue in this case
+				if !strings.Contains(err.Error(), "401") {
```

Comment:
Not silently, there is a difference between “definitely an untagged image” and “server refuses to tell us”.

But returning a typed error from `GetRepositoryTags` so that this wouldn’t have to do a `strings.Contains` would be nice.